katago_server/runs/serializers/run.py

```diff
@@ -21,3 +21,20 @@ class Meta:
             "selfplay_client_config",
             "rating_client_config",
         ]
+
+
+class RunSerializerForClient(HyperlinkedModelSerializer):
+    """
+    Serializer exposing only the fields of a run that a self-play client needs on startup.
+    """
+
+    class Meta:
+        model = Run
+        fields = [
```

Comment:
We should try to unify the code.
Sometimes we use tupple (), sometimes list.

I don't know what is best and why but we should find out and stick to it

Reply:
Based on that https://stackoverflow.com/a/12363023 and https://stackoverflow.com/a/3671640 I think list is fine. We can clena later